lib/web_ui/lib/src/engine/pointer_binding.dart

```diff
@@ -512,19 +588,20 @@ class _PointerAdapter extends _BaseAdapter with _WheelEventListenerMixin {
   }
 
   void _addPointerEventListener(
+    html.EventTarget target,
     String eventName,
     _PointerEventListener handler, {
-    bool acceptOutsideGlasspane = false,
+    bool useCapture = true,
   }) {
-    addEventListener(eventName, (html.Event event) {
+    addEventListener(target, eventName, (html.Event event) {
       final html.PointerEvent pointerEvent = event as html.PointerEvent;
       return handler(pointerEvent);
-    }, acceptOutsideGlasspane: acceptOutsideGlasspane);
+    }, useCapture: useCapture);
   }
 
   @override
   void setup() {
-    _addPointerEventListener('pointerdown', (html.PointerEvent event) {
+    _addPointerEventListener(html.window, 'pointerdown', (html.PointerEvent event) {
```

Comment:
By removing `acceptOutsideGlasspane`, this event will now behave differently. It'll accept `pointerdown` events originating from outside the glasspane (which is undesirable).

Reply:
You're absolutely right - it was a mistake to bind the down event to `window`. I'll correct it shortly.

Reply:
I've changed the target to `glassPaneElement` and rebased the commit.